src/status_im2/contexts/wallet/create_account/edit_derivation_path/style.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.wallet.create-account.edit-derivation-path.style
+  (:require [quo.foundations.colors :as colors]))
+
+(defn screen
+  [top]
+  {:flex       1
+   :margin-top top})
+
+(def header
+  {:padding-horizontal 20
+   :padding-top        12
+   :padding-bottom     8})
+
+(def tag
+  {:padding-horizontal 20
+   :flex-direction     :row})
+
+(def input-container
+  {:padding-horizontal 20
+   :padding-top        12})
+
+(defn save-button-container
+  [bottom]
+  {:flex            1
+   :justify-content :flex-end
+   :padding-bottom  bottom})
+
+(def revealed-address-container
+  {:padding-horizontal 20
+   :padding-top        24})
+
+(def revealed-address
+  {:border-width       1
+   :border-color       colors/success-50-opa-40
```

Comment:
for success it is better to do (`colors/resolve-color :success theme 40`) 
- I need to adjust the colors files so this is a mandatory approach :)

Reply:
Fixed